react/features/authentication/components/web/LoginDialog.tsx

```diff
@@ -315,7 +293,7 @@ function mapStateToProps(state: IReduxState) {
     return {
         _conference: authRequired || conference,
         _configHosts: configHosts,
-        _connecting: connecting || thenableWithCancel,
+        _connecting: Boolean(connecting) || Boolean(thenableWithCancel),
```

Comment:
Did TS complain here?

Reply:
Nope, I just synced the web and mobile to have same checks. Maybe in future PRs we can extract common logic in an abstract or something ...